pkg/kubelet/volumes.go

```diff
@@ -34,8 +34,6 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
-var errUnsupportedVolumeType = fmt.Errorf("unsupported volume type")
-
```

Comment:
this is removed and replaced with a more generic error  createVolumeTypeError - which in theory should never be called from what I can tell, so I was asking if we can remove it all together (errUnsupportedVolumeType was original and I removed in this PR) but replaced with a function to eventually help generate a better error...but as I wondered in the PR I don't know if that error would ever get hit, since if the mounter fails in anyway that error is now surfaced where as before it wasn't...so wonder if we can remove altogether or keep around for some unforeseen scenario??  @swagiaal